addons/account/static/src/components/product_label_section_and_note_field/product_label_section_and_note_field.js

```diff
@@ -135,6 +136,27 @@ export class ProductLabelSectionAndNoteField extends Many2OneField {
                 this.labelNode.el.focus();
             }
         });
+
+        this.onBeforePrint = () => {
+            this.setPrintMode(true);
+        };
+        this.onAfterPrint = () => {
+            this.setPrintMode(false);
+        };
+
+        onMounted(() => {
```

Comment:
We should probably add a small comment to explain why we had to do this hacky'ish fix within the code, to avoid someone to override it at some point.